docs/root/configuration/http_filters/lua_filter.rst

```diff
@@ -10,6 +10,12 @@ Lua
   and verification. This warning will be removed when we feel that the filter has received enough
   testing and API stability to call it generally production ready.
 
+.. attention::
+
+  By default envoy is built without exporting symbols that you may need when interacting with
```

Comment:
Nit: s/envoy/Envoy/ in various palces, s/lua/Lua/, s/bazel/Bazel/ below as well.